modules/administration-guide/partials/proc_viewing-dev-workspace-operator-metrics-on-grafana-dashboards.adoc

```diff
@@ -1,31 +1,30 @@
 [id="proc_viewing-dev-workspace-operator-metrics-on-grafana-dashboards"]
-= Viewing {devworkspace} Operator metrics on Grafana dashboards
+= Viewing {devworkspace} Operator metrics from an OpenShift web console dashboard
 
-To view the {devworkspace} Operator metrics on Grafana with the example dashboard:
+After configuring the in-cluster Prometheus instance to collect {devworkspace} Operator metrics, the metrics can be viewed from a custom dashboard within the *Administrator* perspective of the OpenShift web console.
```

Comment:
Our style guide recommend avoiding passive voice when possible. I've rephrased it to use `you`:
```suggestion
After configuring the in-cluster Prometheus instance to collect {devworkspace} Operator metrics, you can view the metrics on a custom dashboard in the *Administrator* perspective of the OpenShift web console.
```